# This file is no longer needed - we'll use single ObjectPose messages instead
# Keeping for reference but can be deleted
